openstack_exporter/collectors/cinderbackend.py

```diff
@@ -396,6 +271,61 @@ def _report_stats(self, shard_name, backend, data, caps, quota_obj):
                 shard_name, backend, pool_name
             )
 
+    def _report_aggregated_stats(self, backend):
+        pool_name = backend['name']
+        LOG.debug(f"Reporting stats for {backend}/{pool_name}")
```

Comment:
optional: Distinguish from `report_stats` by adding an "aggregate" i.e. "Reporting aggregate stats for".

Reply:
I'm not sure I understand what you are proposing here.  the purpose of this function is to report the aggregate pool stats, which is distinguished from the non aggregate pools (vmware backend vs vmware_fcd)

Reply:
It was about the log line. It's the exact same as in the non-aggregate function. Reading the logs, that would be confusing.